frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/personnel-popup/personnel-popup.component.html

```diff
@@ -0,0 +1,20 @@
+<ng-container *ngIf="personnel$ | async as personnel">
```

Comment:
Dust

You can put the `*ngIf` directly in the first div to reduce nesting.

Reply:
Or should we just use the `<ng-container>` and remove the `<div>`?

Reply:
Wait, I just tested it, the property is not applied when I add it to the `<ng-container>`